go/consensus/tendermint/full/full.go

```diff
@@ -86,6 +86,11 @@ const (
 	// CfgABCIPruneNumKept configures the amount of kept heights if pruning is enabled.
 	CfgABCIPruneNumKept = "consensus.tendermint.abci.prune.num_kept"
 
+	// CfgCheckpointerDisabled disables the ABCI state checkpointer.
+	CfgCheckpointerDisabled = "consensus.tendermint.checkpointer.disabled"
+	// CfgCheckpointerInterval configures the ABCI state checkpointing interval.
```

Comment:
It should be noted that this is the check interval (e.g., how often to check whether a new checkpoint needs to be taken) not the checkpoint interval.